spring-boot-admin-server-ui-javamelody/pom.xml

```diff
@@ -0,0 +1,108 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>de.codecentric</groupId>
+		<artifactId>spring-boot-admin</artifactId>
+		<version>1.5.1-SNAPSHOT</version>
+		<relativePath>..</relativePath>
+	</parent>
+
+	<artifactId>spring-boot-admin-server-ui-javamelody</artifactId>
+
+    <properties>
+        <javamelody.version>1.67.0</javamelody.version>
+        <ehcache.version>2.6.0</ehcache.version>
+    </properties>
+
+
+    <dependencies>
+        <dependency>
+            <groupId>de.codecentric</groupId>
+            <artifactId>spring-boot-admin-server</artifactId>
+        </dependency>
+		<dependency>
+			<groupId>de.codecentric</groupId>
+			<artifactId>spring-boot-admin-server-ui</artifactId>
+			<scope>runtime</scope>
+		</dependency>
+		<dependency>
+			<groupId>net.bull.javamelody</groupId>
+			<artifactId>javamelody-core</artifactId>
+			<version>${javamelody.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>net.sf.ehcache</groupId>
+			<artifactId>ehcache-core</artifactId>
+			<version>${ehcache.version}</version>
+		</dependency>
```

Comment:
I suggest we add this in the documentation instead of the pom.xml so users can choose for themselves if they want javamelody reporting to work. Personally I don't use it and rather don't have a transitive log4j/itext dependency

Reply:
No, I think that many users would be really happy to have the pdf reports. And, adding a iText dependency would be debatable in a monitored app but this dependency (with exclusions on transitive dependencies) is in my opinion not a problem in spring-boot-admin. So I think that we should really keep the iText dependency like it is now.
For Log4J, it is not included at the moment and finally it seems not needed since there is already a log4j-over-slf4j dependency in spring-boot-admin.